lib/core/src/animation/animator/fixed_step.rs

```diff
@@ -15,11 +15,14 @@ use crate::system::web::animation_frame_loop::AnimationFrameLoop;
 /// This struct counts the intervals in a time period.
 #[derive(Debug)]
 pub struct IntervalCounter {
+    /// Interval duration.
     pub interval_duration : f32,
+    /// The amount of time accumulated by `IntervalCounter`'s `add_time`.
     pub accumulated_time  : f32
 }
```

Comment:
We have not been talking about it earlier, but I believe that in case we document pub fields, they should not be aligned and they should be separated vertically. What do you think about this instead? CC @farmaazon 

```rust
pub struct IntervalCounter {
    /// Interval duration.
    pub interval_duration: f32,

    /// The amount of time accumulated by `IntervalCounter`'s `add_time`.
    pub accumulated_time: f32
}
```

Reply:
I thought about doing it, it feels more readable to me. I will change it accordingly